vendor/extensions/articles/db/seeds.rb

```diff
@@ -20,31 +20,33 @@
 
 Refinery::Articles::Article.destroy_all
 
-Refinery::Articles::Article.create(
+if Rails.env.downcase != "production"
```

Comment:
http://www.rubydoc.info/github/rails/rails/Rails.env

yea it looks like you don't need it